Make_It_Talk/say-temp.py

```diff
@@ -34,15 +40,15 @@ def play_file(wavfile):
     wavfile = "/numbers/" + wavfile
     print("Playing", wavfile)
     with open(wavfile, "rb") as f:
-        wav = audioio.WaveFile(f)
+        wav = audiocore.WaveFile(f)
         aout.play(wav)
         while aout.playing:
             pass
 
 # Function should take an integer -299 to 299 and say it
 # Assumes wav files are available for the numbers
 def read_temp(temp):
-    play_file("temperature.wav")
+    play_file("The temperature is.wav")
```

Comment:
When did the filename change? Filenames with spaces can be awkward.

Reply:
If you look in the Make_It_Talk/numbers directory, temperature.wav doesn't exist, which causes the code to fail, and I'm like 99% certain that it was intended to be "The temperature is.wav" instead. Alternatively, I could rename it to temperature.wav, but I have tested it and it does work.

Reply:
Ok, this is fine. Thanks!